README.md

```diff
@@ -1,13 +1,14 @@
 # Project Express API
 
-Replace this readme with your own information about your project.
+BooksData API built in Node using Express. The task was to create an API with different endpoints. The API uses a static data set stored as a JSON-file.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+My Routes:
+        "/": "Welcome Page to API BookData",
+	 		  "/books": "All books",
+        "/books/author": "authors in alphabetic order",
+        "/books/:id/": "Search books by BookID",
```

Comment:
Hi Naghemeh,

Good information. I would even recommend to include this on the start page.